x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_summary_charts_panel/translations.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+
+export const CHARTS_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.alerts.charts.chartsTitle',
+  {
+    defaultMessage: 'Charts',
+  }
+);
+
+export const SEVERITY_LEVELS_INSPECT_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionResponse.alertsBySeverity.inspectModalSeverityTitle',
+  {
+    defaultMessage: 'Severity levels',
+  }
+);
+
+export const SEVERITY_LEVELS_TITLE = i18n.translate(
```

Comment:
Can this and `SEVERITY_LEVELS_INSPECT_TITLE` be merged to just be `.alertsBySeverity.severityTitle'` and used in both places?